### PR TITLE
Update script to account for latest build installer change

### PR DESCRIPTION
**Pull request by musm**

### CODE REVIEW

**Comment by musm** [comment]
Not sure why this is stalling? I downloaded the nightlies and used the flags here and it worked.

**Comment by musm** [comment]
Is this  because appveyor is using the old build script?

File: bin/install.ps1 [hunk 1]
```diff
@@ -19,7 +19,7 @@ if ($env:PLATFORM -eq "x86") {
 
 if ($env:JULIA_VERSION -in 'latest', 'nightly') {
     # TODO: move to nightly to be consistent with Travis and download page
-    $julia_version = [Version]"0.7"
+    $julia_version = [Version]"1.4"
```
Comment (musm) [inline bin/install.ps1:22]: Not sure why the version here is hardcoded?  Anyways, separate issue.

**Comment by musm** [comment]
@simonbyrne   this fixes latest appveyor failures due to the build changes. Should be ready on my end. Please don't forget to squash. Thanks.

File: bin/install.ps1 (hunk 1 above)
Comment (simonbyrne) [reply bin/install.ps1:22]: It only affects the choice of commands used to trigger tests, e.g. https://github.com/JuliaCI/Appveyor.jl/blob/178812944acf5b2020040c779423a3d771d3a9b7/bin/install.ps1#L59.

Comment (musm) [reply bin/install.ps1:22]: Got it.  Had to update it to 1.4 so  versioning uses the correct command for the nightly builds, in the changes made in this PR.

**Comment by simonbyrne** [comment]
Maybe add 1.3 to the test matrix?

**Comment by simonbyrne** [comment]
and 1.2 as well?

**Comment by musm** [comment]
Drop 0.6?

**Comment by simonbyrne** [comment]
If it still works, no harm in keeping it around for now.

**Comment by simonbyrne** [comment]
Thanks!

**Pull request merged**